emails/app/helpers/spree/mail_helper.rb

```diff
@@ -30,5 +30,9 @@ def logo_path
 
       return main_app.cdn_image_url(store_logo) if store_logo.image?
     end
+
+    def full_asset_url(path)
```

Comment:
@aleksandarpetrushev why do we need a custom method when there is one already present in rails?

https://api.rubyonrails.org/v6.1.4/classes/ActionView/Helpers/AssetUrlHelper.html#method-i-asset_url

Reply:
I am using it in this method. Because it returns only the "/assets/..." part in emails apparently. So for example in views returns the full asset url, and in the emails it does not include the host part

Reply:
@aleksandarpetrushev `asset_url` generates and URL with the host included, you can not just combine strings to "fix" this

Reply:
<img width="632" alt="Screenshot 2021-12-15 at 13 40 51" src="https://user-images.githubusercontent.com/20384428/146188495-27a7c52b-3a2e-4cc0-87a7-473c3a9ca735.png">
This is what image_url('noimage/small.png') (or asset_url(..)) returns in the email.